vlm/src/main/scala/geotrellis/contrib/vlm/gdal/VSIPath.scala

```diff
@@ -0,0 +1,208 @@
+package geotrellis.contrib.vlm.gdal
+
+import scala.util.matching.Regex
+
+import java.io.File
+
+
+case class VSIPath(
+  path: String,
+  compressedFileDelimiter: String = "!"
```

Comment:
When can it be something other than `"!"` ?

Reply:
That's there to catch the cases where there's a `!` in one of the file names for some reason.